$NetBSD: patch-ac,v 1.1 2005/10/22 17:56:31 rillig Exp $

The IRIX C compiler does not allow non-constant initializers for struct
members.

--- system/X11/display_list.c.orig	Thu Jan 11 19:14:30 1996
+++ system/X11/display_list.c	Sat Oct 22 19:49:39 2005
@@ -149,7 +149,7 @@ void disp_config(void)
    XtResource mailfile = {"inbox", "Inbox", XtRString,
                           sizeof(str), 0, XtRString, mailfile_str};
    XtResource monoflag = {"monochrome", "Monochrome", XtRBoolean,
-                          sizeof(mono), 0, XtRBoolean, &mono};
+                          sizeof(mono), 0, XtRBoolean, NULL};
    XtResource frmcount = {"frameCount", "FrameCount", XtRInt,
                           sizeof(frames), 0, XtRInt, &frames};
 
@@ -164,6 +164,7 @@ void disp_config(void)
    strcpy(command_str, str);
    XtGetApplicationResources(toplevel, &str, &mailfile, 1, NULL, 0);
    strcpy(mailfile_str, str);
+   monoflag.default_addr = &mono;
    XtGetApplicationResources(toplevel, &mono, &monoflag, 1, NULL, 0);
    if (mono) color_mode = 0;
    
